post-scan-actions/aws-python-publish-malware-to-sns/Makefile

```diff
@@ -0,0 +1,62 @@
+.PHONY: zip
+
+MKDIR := mkdir
+ZIP := zip
+
+zip:
+	$(MKDIR) -p zip
+	$(ZIP) zip/send-malware-to-sns.zip handler.py
+
+create-target-sns-topic:
+	aws sns create-topic --name $(TARGET_SNS_TOPIC_NAME)
+
+create-publish-malware-policy:
+	aws iam create-policy \
+	--policy-name $(PUBLISH_MALWARE_POLICY_NAME) \
+	--policy-document "{\"Version\":\"2012-10-17\",\"Statement\":[{\"Sid\":\"VisualEditor0\",\"Effect\":\"Allow\",\"Action\":[\"sns:TagResource\",\"sns:Publish\",\"account:GetAlternateContact\"],\"Resource\":\"*\"}]}"
+
+create-publish-malware-role:
+	aws iam create-role \
+	--role-name $(PUBLISH_MALWARE_ROLE_NAME) \
+	--assume-role-policy-document "{\"Version\": \"2012-10-17\",\"Statement\": [{\"Action\": \"sts:AssumeRole\",\"Effect\": \"Allow\",\"Principal\": {\"Service\": \"lambda.amazonaws.com\"}}]}"
+
+attach-policy:
+	aws iam attach-role-policy \
+	--role-name $(PUBLISH_MALWARE_ROLE_NAME) \
+	--policy-arn arn:aws:iam::aws:policy/service-role/AWSLambdaBasicExecutionRole
+
+	aws iam attach-role-policy \
+	--role-name $(PUBLISH_MALWARE_ROLE_NAME) \
+	--policy-arn $(PUBLISH_MALWARE_POLICY_ARN)
+
+create-function: zip
+	aws lambda create-function \
+	--function-name $(PUBLISH_MALWARE_LAMBDA_NAME) \
+	--role $(PUBLISH_MALWARE_ROLE_ARN) \
+	--runtime python3.9 \
+	--timeout 30 \
```

Comment:
They are not matched with ones in the template. Why?

Reply:
Will align them in the next commit.

Reply:
The runtime is not aligned still.

Reply:
Sorry for the lost, fixed already.